### DISCUSSION TITLE
Possible to export and import an object's parameters?

### DISCUSSION THREAD

**Discussion by GrantWChapman** [exult/exult#539] [General]
Is it possible to export and import an object's parameters in Exult Studio. Things like the in hand offsets and flags like if an object is a weapon and has paperdoll art?
I ask because it's getting kinda tedious working with 
in hand offsets for weapon sprites and thought it would help if I could import that data instead of having to type it for every frame.

**Comment by agentorangeguy** [comment]
I don't think so but this would be a great thing to add!

**Comment by KnightCaptainU7** [comment]
I third this request, so I opened a Feature Request for it.
https://github.com/exult/exult/issues/588

If we were to prioritize the specifics, would in-hand offsets be the top priority? Or all X,Y values?
